qt/scientific_interfaces/Inelastic/Common/InterfaceUtils.cpp

```diff
@@ -0,0 +1,168 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "InterfaceUtils.h"
+#include "MantidKernel/Logger.h"
+#include <QDomDocument>
+#include <QFile>
+#include <QMessageBox>
+#include <QtXml>
+#include <boost/algorithm/string.hpp>
+#include <boost/numeric/conversion/cast.hpp>
+
+namespace {
+Mantid::Kernel::Logger g_log("InterfaceUtils");
+}
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace InterfaceUtils {
+
+/**
+ * General functions used in inelastic/indirect tabs.
+ */
+static QStringList toQStringList(std::vector<std::string> const &strings) {
+  QStringList list;
+  for (auto const &str : strings)
+    list << QString::fromStdString(str);
+  return list;
+}
```

Comment:
I think we can remove this function altogether and just use this
https://github.com/mantidproject/mantid/blob/3e23675f40cc3ddb1bd8fe46c73efaf9aab61a3b/qt/widgets/common/src/ParseKeyValueString.cpp#L275